src/metrics.py

```diff
@@ -144,10 +145,15 @@ def run_command(command: str) -> dict:
     Execute command via the command line and load the output into dictionary.
     """
     with timy.Timer(command):
+        # temporary workaround for https://github.com/borgbackup/borg/issues/7255
+        tmp_env=os.environ.copy()
+        tmp_env["HOME"]="/tmp/borgmatic-exporter-cache"
+        tmp_env["BORG_UNKNOWN_UNENCRYPTED_REPO_ACCESS_IS_OK"]="yes"
```

Comment:
Maybe it's better to set this sort of variable based on the command line flag. To make it a consious decision.

Reply:
I would wait with that. It worked first... but now my container started again to throw the error... 🙄 
May need more investigation seems that borgmatic does not pass "--bypass-lock down to borg", but not sure why it seemed to work first. Sounds more that I messed something up there.